Adapter_oM/Settings-Config/AdapterSettings.cs

```diff
@@ -61,6 +61,9 @@ public class AdapterSettings : IObject
         public virtual bool UseAdapterId { get; set; } = true;
         public virtual bool UseHashComparerAsDefault { get; set; } = false;
         public virtual bool ProcessInMemory { get; set; } = false;
+        [Description("If true, Objects found to be the same acording to the AdapterComparer for the provided type is checked for full equality using the HashComparer of the type, and is not updated if equal acording to the HashComaprer.\n" + 
+                     "If false, Objects found identical according to the AdapterComparer for the provided type is always sent to be updated.")]
```

Comment:
```suggestion
        [Description("If true, Objects found to be the same according to the AdapterComparer for the provided type are checked for full equality using the HashComparer of the type (which by default checks every property except BHoM_Guid). If equal according to the HashComparer, they are not updated.\n" + 
                     "Otherwise, Objects found identical according to the AdapterComparer for the provided type are sent to the Update method.")]
```